frontend/src/graphql/queries/product/product-query.ts

```diff
@@ -0,0 +1,25 @@
+import { gql } from '@apollo/client'
+
+export const GET_ALL_PRODUCTS = gql`
+  query getAllProducts {
```

Comment:
You should use a fragment to avoid repeating the same thing right below it

Reply:
done